### PR TITLE
fix(ci): Make nightly builds actually work

### PR DESCRIPTION
**Pull request by sjbarag**
Looks like the issue was mostly me manually setting CI=true via the `.env` block?  :shrug:

### CODE REVIEW

File: .github/workflows/nightly-build.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   name: "Nightly build",
   on: {
     schedule: [
-      { cron: "20 17 * * *" }
+      { cron: "05 7 * * *" }
```
Comment (alimnios72) [inline .github/workflows/nightly-build.yaml:5]: Does the `5` need to be prefixed with `0`? Also, in which timezone does this run?

File: .github/workflows/nightly-build.yaml [hunk 2]
```diff
@@ -14,29 +14,18 @@
           name: "Set up node 12",
           uses: "actions/setup-node@v1",
           with: {
-            registry-url: "https://registry.npmjs.org",
+            registry-url: "https://registry.npmjs.com",
```
Comment (alimnios72) [inline .github/workflows/nightly-build.yaml:17]: Why do we switch from `.org` to `.com`?

File: .github/workflows/nightly-build.yaml (hunk 1 above)
Comment (sjbarag) [reply .github/workflows/nightly-build.yaml:5]: I suppose I should either do both or neither, huh?  😅 

Also, UTC!

File: .github/workflows/nightly-build.yaml (hunk 2 above)
Comment (sjbarag) [reply .github/workflows/nightly-build.yaml:17]: Our `.npmrc` specifies `.com` so I went for consistency 🤷

Comment (alimnios72) [reply .github/workflows/nightly-build.yaml:17]: That makes sense

**Pull request merged**